crypto.go

```diff
@@ -799,6 +799,28 @@ func (s chachaPolyScheme) NonceSize() int {
 	return chacha20poly1305.NonceSize
 }
 
+//////////
+// Export-only AEAD scheme
+
+type exportOnlyScheme struct {
+}
+
+func (s exportOnlyScheme) ID() AEADID {
+	return AEAD_EXPORT_ONLY
+}
+
+func (s exportOnlyScheme) New(key []byte) (cipher.AEAD, error) {
+	panic("Not supported")
```

Comment:
You should test these and verify that they throw.

Reply:
Good call. I'll add a test for that.